### PR TITLE
Adding template for Juniper ACX7024 router and ZPE NSR-BASE-DAC with preloaded expansion modules

### PR DESCRIPTION
**Pull request by modonovan**
The purpose of this pull request is to add a new template, not yet created, for the [Juniper ACX7024 router](https://www.juniper.net/us/en/products/routers/acx-series/acx7000-family-cloud-metro-routers/acx7000-line/acx7024-acx7024x-cloud-metro-routers-datasheet.html) as well as the [ZPE Network Services Router with preloaded expansion modules](https://zpesystems.com/products/nodegrid-services-router/).

In the case of the Juniper ACX7024, I have modeled this to the best of my capability based on the Juniper product datasheet for this router and in conjunction with the ACX7100 templates that preexist. 

In the case of the ZPE NSR device, the template is modeled on the assume that;

* There is a [16 port serial console module](https://zpesystems.com/products/nodegrid-services-router/serial-expansion-card/) in Slot 1 of the device (hence the numbering ttyS1-1 etc)
* There is a [16 port ethernet switch module](https://zpesystems.com/products/nodegrid-services-router/ethernet-expansion-card/) in Slot 2 of the device (hence the numbering starting netS2-1 etc)

### CODE REVIEW

File: tests/known-modules.pickle
Comment (DanSheps) [inline tests/known-modules.pickle:1]: Remove from commit please

File: tests/known-slugs.pickle
Comment (DanSheps) [inline tests/known-slugs.pickle:1]: Remove from commit please

**Review by DanSheps — CHANGES_REQUESTED** [review]

File: tests/known-modules.pickle
Comment (modonovan) [reply tests/known-modules.pickle:1]: I'm a relative newbie to submitting pull requests on github.com so excuse my ignorance. I'm not sure I know how to resolve this. I've installed `gh` locally and issued a `gh pr checkout 1955`. I've then removed the files in question and did a git push. Is there something obvious I'm missing here?

File: tests/known-slugs.pickle
Comment (modonovan) [reply tests/known-slugs.pickle:1]: Resolved by NEW PR !2011

File: tests/known-modules.pickle
Comment (modonovan) [reply tests/known-modules.pickle:1]: Resolved by NEW PR !2011

**Pull request closed without merging**